packages/blockchain-api/app.alfajores.yaml

```diff
@@ -5,6 +5,10 @@ env_variables:
   DEPLOY_ENV: "alfajores"
   EXCHANGE_RATES_API: "https://api.exchangeratesapi.io"
   BLOCKSCOUT_API: "https://alfajores-blockscout.celo-testnet.org/api"
+  # Pull addresses from the build artifacts of the network in protocol/build
+  CELO_GOLD_ADDRESS: "0xB8641365dBe943Bc2fb6977e6FBc1630EF47dB5a"
```

Comment:
These addresses can be pulled in from the hosted node specified below, and there's a stackdriver uptime check to ensure that it's already live. Any reason to switch back to hardcoding addresses?

Reply:
Nope! I'll revert this, I think we just changed it out of habit